app/models/course/assessment/assessment_ability.rb

```diff
@@ -73,7 +73,8 @@ def allow_read_material
   def allow_create_assessment_submission
     can :create, Course::Assessment::Submission,
         experience_points_record: { course_user: { user_id: user.id } }
-    can [:update, :generate_live_feedback], Course::Assessment::Submission, assessment_submission_attempting_hash(user)
+    can [:update, :generate_live_feedback, :save_live_feedback], Course::Assessment::Submission,
```

Comment:
do this change on the commit where you firstly create the definition of the function `save_live_feedback`

Reply:
done